domains/cola.json

```
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "randomboi404",
+        "email": "aauispwujd@gmail.com",
+        "discord": "colaaaaaaaaaaaaa"
+    },
+    "records": {
+        "MX": ["mx1.improvmx.com", "mx2.improvmx.com"],
+        "TXT": ["v=spf1 include:spf.improvmx.com ~all"]
+    },
+    "proxied": true
```

You can't proxied if its just Email purposes
```suggestion
    }
```